cpp/kiss_icp/core/Preprocessing.cpp

```diff
@@ -55,7 +55,8 @@ std::vector<Eigen::Vector3d> VoxelDownsample(const std::vector<Eigen::Vector3d>
     }
     std::vector<Eigen::Vector3d> frame_dowsampled;
     frame_dowsampled.reserve(grid.size());
-    for (const auto &[_, point] : grid) {
+    for (const auto &[voxel, point] : grid) {
+        (void)voxel;
```

Comment:
Sadly on older gcc this will fail with unused-variable' This is the case for ROS-melodic, and judging by how our community works, I know that if I break the build of ROS-melodic we will get issues automatically :) So I guess is worth the hack